lib/validation-stream.js

```diff
@@ -18,11 +24,17 @@ function ValidationStream(options) {
   // Create internal placeholder for CSS
   this._css = [];
 
-  // Generate our request
-  this.generateRequest();
+  if (isBlank(this.options.text)) {
+    // The W3C service only understands nonempty CSS. Fork a dummy event to be acted on after the
+    // client has yielded.
+    setTimeout(this._emitEmptyCssEvents.bind(this), 0);
```

Comment:
I really appreciate you taking the time to share this tip! I'm still ramping up on Node.js (and JavaScript in general) so your feedback is dear!